docs/meshstack.aks-platform-quickstart-guide.md

```diff
@@ -0,0 +1,223 @@
+---
+id: meshstack.aks-platform-quickstart-guide
+title: AKS Platform Quickstart Guide
+---
+
+
+# Introduction
+
+Welcome to the **First Platform: AKS guide**! This guide provides step-by-step instructions to help you quickly set up
+your first meshStack platform integration with Azure Kubernetes Service (AKS).
+
+# Goals
+
+By the end of this guide, you will have achieved:
+
+1. Creating a platform integration with AKS. This will allow:
+   - provisioning new namespaces
+   - assign users
+   - assign tags (optional)
+2. Successfully testing the AKS integration
+
+This allows your internal users to open up meshStack and provision themselves namespaces, including identity & access management.
+
+# Prerequisites
+
+Before starting, ensure you have the following:
+
+- An AKS Cluster that is publicly accessible via the internet
+- A place to run Terraform. For example, your local machine, or the Azure cloud shell.
+
+# Preparing Infrastructure in Azure
+
+To integrate your AKS cluster into meshStack properly, we require a few things to be in place in AKS & Azure for a successful integration.
+
+In this guide, we will use our Terraform module to set these things up. You can also do this manually, but we highly recommend doing this with Terraform, and for this guide we assume you did.
+
+Amongst other things, the Terraform module will create a service user with the right permissions in the Kubernetes cluster and create a service principal in Azure to interact with your Entra tenant.
+
+Click [here](https://github.com/meshcloud/terraform-aks-meshplatform) and follow along the guide in the README to use the Terraform module.
+
+Make sure you execute the last step of exporting the outputs and save these somewhere. We need them later in this guide.
+
+# Creating the Platform
+
+1. **Open the platform list in the service management area**
```

Comment:
```suggestion
1. **Open the platform list in the service management area**
```
Starting with the release today this will be called platform builder